apps/token-manager/app/src/App.js

```diff
@@ -212,5 +204,10 @@ export default () => {
   const theme = useTheme()
   const { layoutName } = useLayout()
 
+  const themeMode = useThemeMode()
```

Comment:
Should we remove this for now?

Reply:
Oops this was for a local demo, it shouldn’t have been committed :o

Reply:
Done https://github.com/aragon/aragon-apps/pull/1046/commits/03b2aaa47db934e20f206ed73c66269fc29369d1